src/tags/jsonkeys.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * @Author: RagingLink
+ * @Date: 2020-07-17 19:24:32
+ * @Last Modified by: RagingLink
+ * @Last Modified time: 2020-09-11 21:25:15
+ *
+ * This project uses the AGPLv3 license. Please read the license file before using/adapting any of the code.
+ */
+
+const Builder = require('../structures/TagBuilder');
+
+module.exports =
+    Builder.ArrayTag('jsonkeys')
+        .withAlias('jkeys')
+        .withArgs(a => [a.require('object'), a.optional('path')])
+        .withDesc('Retrieves all keys from provided JSON object.' + 
+        '`object` can be a JSON object, array, or string. If a string is provided, a variable with the same name will be used.\n' +
```

Comment:
You should review this description because a JSON object is by definition the notation of a Javascript object as a string. This will lead to confusion for advanced developers or people used to bbtag being a language without "real types"

Reply:
I think it's fine, in this context it needs to be an object in JSON notation (ex. quotes keys, double quotes, etc.)